index.html

```diff
@@ -0,0 +1,995 @@
+                <div class="second__heores-itext position-absolute w-auto"><a href="#" class="decoration-none text-dark">Pandemic Heroes</a></div>
+              </div>
+              <div class="second__heroes-tcontainer pt-3">
+                <h2><a href="#" class="decoration-none text-dark font-weight-light">Restauranteur Delivers Oranges to Hospital Workers Amid Coronavirus Crisis</a></h2>
+                <span class="second__heroes-t">"We all have to help each other," Matthew Fermin, owner of East Harlem restaurant Santiago's Beer Garden, told Newsweek.</span>
+              </div>
+            </div>
+            <div class="col-sm-4 col-12 pr-1 pl-1">
+              <div class="second__heores-icontainer position-relative">
+                <a href="#"><img src="https://d.newsweek.com/en/full/1578426/zoe-monterola-founder-six-feet-supplies.jpg?w=397&h=397&f=f7a624e81aa902878bb39d5fe83e971f" alt="Zoe Monterola founder of Six Feet Supplies"></a>
+                <div class="second__heores-itext position-absolute w-auto"><a href="#" class="decoration-none text-dark">Pandemic Heroes</a></div>
+              </div>
+              <div class="second__heroes-tcontainer pt-3">
+                <h2><a href="#" class="decoration-none text-dark font-weight-light">Teenagers Shop for the Vulnerable During Coronavirus Emergency</a></h2>
+                <span class="second__heroes-t">"I almost cried," said a user of Six Feet Supplies, a charity started by teenagers who shop for those susceptible to COVID-19.</span>
+              </div>
+            </div>
+            <div class="col-sm-4 col-12 pr-0 pl-3">
+              <div class="second__heores-icontainer position-relative">
+                <a href="#"><img src="https://d.newsweek.com/en/full/1578399/coronavirus-pandemic-heroes-food-nurses-philadelphia.jpg?w=397&h=397&f=df54d52fbce46b666dc3c098564d1f08" alt="coronavirus pandemic heroes food nurses philadelphia "></a>
+                <div class="second__heores-itext position-absolute w-auto"><a href="#" class="decoration-none text-dark">Pandemic Heroes</a></div>
+              </div>
+              <div class="second__heroes-tcontainer pt-3">
+                <h2><a href="#" class="decoration-none text-dark font-weight-light">Philly DJ Rallies Friends, Strangers to Feed Nurses and Support Businesses </a></h2>
+                <span class="second__heroes-t">"I want to keep it going as long as we can," Ryan O'Connell told Newsweek.</span>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="first__column-title text-uppercase font-weight-bold mt-3">Culture & Travel</div>
+        <div class="container">
+          <div class="row">
+            <div class="col-sm-4 col-12 pr-3 pl-0">
+              <div class="second__heores-icontainer position-relative">
```

Comment:
[Required] on a small screen, Culture and travel section should be one column layout but currently, it is three columns layout. 
in your project :point_down: 
![Screenshot from 2020-04-17 09-50-08](https://user-images.githubusercontent.com/37747619/79533730-55bc2780-8098-11ea-8344-8fb31f4963da.png)

it should be :point_down: 
![Screenshot from 2020-04-17 09-50-48](https://user-images.githubusercontent.com/37747619/79533756-68366100-8098-11ea-9443-02f3d101bb16.png)

Reply:
Make a similar layout for the heroes of the pandemic section also.